spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1018,21 +1020,28 @@ public String getSparkUIUrl() {
   }
 
   public void populateSparkWebUrl(InterpreterContext ctx) {
-    if (sparkUrl == null) {
-      sparkUrl = getSparkUIUrl();
-      Map<String, String> infos = new java.util.HashMap<>();
-      if (sparkUrl != null) {
-        infos.put("url", sparkUrl);
-        if (ctx != null && ctx.getClient() != null) {
-          logger.info("Sending metainfos to Zeppelin server: {}", infos.toString());
-          getZeppelinContext().setEventClient(ctx.getClient());
-          ctx.getClient().onMetaInfosReceived(infos);
-        }
+    sparkUrl = getSparkUIUrl();
+    Map<String, String> infos = new java.util.HashMap<>();
+    infos.put("url", sparkUrl);
+    java.lang.Boolean uiEnabled = java.lang.Boolean.parseBoolean(
+            property.getProperty("spark.ui.enabled", "true"));
```

Comment:
Ok, I can trim here as well as I also need to rebase to fix some merge conflicts. parseBoolean already deals with case sensitive, so only trim() is needed.